### PR TITLE
0.10.0

### PR DESCRIPTION
**Pull request by tbchrist**
Additions:
 * Multiple sharding pool support
 * SSL replication support
 * Stream encryption support
 * Post-import index creation
 * Miscellaneous bugfixes

### CODE REVIEW

**Comment by komapa** [comment]
:bullettrain_front: LGTM

File: lib/jetpants/host.rb [hunk 1]
```diff
@@ -214,6 +214,19 @@ def fast_copy_chain(base_dir, targets, options={})
       else
         output "Compression disabled -- no compression method specified in Jetpants config file"
       end
+
+      should_encrypt = false
+      targets.each do |t|
+        should_encrypt = should_encrypt || should_encrypt_with?(t)
+      end
+
+      if Jetpants.encrypt_with && Jetpants.decrypt_with && should_encrypt
```
Comment (spacejam) [inline lib/jetpants/host.rb]: If should_encrypt is true, but an encrypt_with/decrypt_with has not been provided, is it desirable to silently degrade the crypto?

Comment (tbchrist) [inline lib/jetpants/host.rb]: To be fair, we don't silently downgrade, there is an output line below.  That being said, it may be worth adding a prompt.

**Comment by spacejam** [comment]
ship it! :syringe: :rage4: :gun:

**Pull request merged**